### PR TITLE
Update readme structure

### PR DESCRIPTION
**Pull request by scopenco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
-- nginx
+## Requirements
+
+### Chef Client
+
+* Chef Client 12.+
+
+### Platforms
+
+* Ubuntu >= 12.04
```
Comment (tas50) [inline README.md]: This should be 14.04+ since 12.04 is EOL by 6 months

File: README.md [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
+* Debian >= 7
+* CentOS >= 6
+
+### Cookbooks
+
+* [apt](https://supermarket.chef.io/cookbooks/apt)
+* [yum](https://supermarket.chef.io/cookbooks/yum)
+* [chef_nginx](https://supermarket.chef.io/cookbooks/yum)
```
Comment (tas50) [inline README.md]: The yum link made it in twice

**Comment by scopenco** [comment]
Fixed

**Comment by tas50** [comment]
Thanks

**Pull request merged**